src/app/containers/HomePage/style.scss

```diff
@@ -8,51 +8,298 @@
   .content {
     width: $content-width;
     margin: 48px auto 0 auto;
-    padding-bottom: 30px;
-    box-shadow: 0px 0px 12px rgba(188, 188, 188, 0.25);
-    background-color: #fff;
     color: rgba(0, 0, 0, 0.6);
     overflow: auto;
+    background-color: #f5f8fa;
   }
-  .title {
+  .notification {
+    background-color: white;
     width: 100%;
-    height: 43px;
-    margin-top: 36px;
-    float: left;
-    padding-left: 36px;
-    font-weight: 600;
-    font-size: 34px;
-    line-height: 43px;
-    color: #1c83c6;
-
+    height: 90px;
+    padding: 24px 22px;
+    display: flex;
+    font-size: 16px;
+    margin-bottom: 24px;
+    border-radius: 4px;
+    box-shadow: 0px 0px 12px rgba(188, 188, 188, 0.25);
+    color: rgba(0, 0, 0, 0.8);
+    h3 {
+      font-weight: 700;
+      line-height: 25px;
+    }
+    p {
+      font-weight: 600;
+      line-height: 20px;
+    }
```

Comment:
don't nested the style, give it specific name like `notificationText` `notificationTitle` `notificationMessage` `notificationDescription` etc...